### PR TITLE
Followers

### PR DESCRIPTION
**Pull request by GaganDureja**

### CODE REVIEW

File: templates/users/show.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+<div>
+  <h6>First Name - {{ user.first_name }}</h6>
+  <h6>Last Name - {{ user.last_name }}</h6>
+  <h6>Followers - {{ user.followers.count }}</h6>
+  {% if request.user|can_follow:user %}
+    <a href="javascript:void(0)" onclick="follow('POST')">Follow</a>
+  {% endif %}
+  {% if request.user|can_unfollow:user %}
+  <a href="javascript:void(0)" onclick="follow('DELETE')">Unfollow</a>
+  {% endif %}
```
Comment (grsahil20) [inline templates/users/show.html:15]: these should be just `is_following` and then you can do 
`request.user != user and request.user|is_following:user`
`request.user != user and !request.user|is_following:user`

File: users/templatetags/follower_tags.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+@register.filter
+def can_follow(user, follow_to):
+  if user.id == follow_to.id:
+    return False
+  if user in follow_to.followers.all():
+    return False
+  return True
+
+@register.filter
+def can_unfollow(user, follow_to):
```
Comment (grsahil20) [inline users/templatetags/follower_tags.py:14]: `def can_unfollow(following_user, followed_user):`

File: templates/users/show.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+</div>
+{% endblock %}
+
+{% block specific_js %}
+  <script>
+    function follow(request_method) {
+      console.log("{{user.id}}");
+      fetch(`{% url 'users:ManageFollower' user.id %}`, {
```
Comment (grsahil20) [inline templates/users/show.html:22]: I told you to pass url and method from link. 

That way this ajax can be used for any  POST and DELETE

**Review by grsahil20 — COMMENTED** [review]
👍

File: templates/users/show.html (hunk 3 above)
Comment (GaganDureja) [reply templates/users/show.html:22]: okay my mistake let me make a new common function for all. please confirm for updating data

**Review by grsahil20 — APPROVED** [review]
I have not tested but looks good

**Pull request merged**